pkg/util/util.go

```diff
@@ -1031,6 +1031,14 @@ func IsAWSPlatform() bool {
 	return isAWS
 }
 
+func IsHCIPlatform() bool {
```

Comment:
Add comment

Reply:
hci is a generic term, better rename to something specific like IsFusionHCIWithScale

Reply:
Make sense.